src/core/Indentation.js

```diff
@@ -26,7 +23,7 @@ export default class Indentation {
      * @return {String}
      */
     getIndent() {
-        return repeat(this.indent, this.indentTypes.length);
+        return this.indent.repeat(this.indentTypes.length);
```

Comment:
`.repeat` is supported by modern browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat